### ISSUE TITLE
Maximum number of lines in the buffer

### ISSUE THREAD

**Issue by cipriancraciun** [dhamidi/tmenu#14]
It would be nice to have a configuration option that tels `tmenu` how many lines it should allow to buffer.

Moreover when this limit is reached, it should display only once the overflow warning.  (Currently it prints it for each extra line.)

Furthermore although it ignores the extra lines, the displayed status bar still counts them.
